app/models/adjustment.rb

```diff
@@ -0,0 +1,30 @@
+class Adjustment < ApplicationRecord
+  delegate :value, to: :financial_transaction
+  delegate :financial_quarter, to: :financial_transaction
+  delegate :financial_year, to: :financial_transaction
+
+  belongs_to :financial_transaction,
```

Comment:
good question. I *think* that where we're headed is a 'base' Transaction model which is used by various financial classes:

- `Actual`
- `Adjustment`
- `Refund`

i.e. that transaction is a technical term which can be limited to the model layer so that the user interface, controllers, and service objects, importers, exporters etc can all think in terms of `Actual`, `Refund` and `Adjustment`. The fact that the common financial attributes are delegated to `Transaction` are implementation details at the model layer.